src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ParameterSymbolReferenceFinder.cs

```diff
@@ -43,54 +43,7 @@ protected override bool CanFind(IParameterSymbol symbol)
             CancellationToken cancellationToken)
         {
             return FindReferencesInDocumentUsingIdentifierAsync(
-                symbol, symbol.Name, state,
-                GetParameterSymbolsMatchFunction(symbol),
-                cancellationToken);
-        }
-
-        private static Func<ISymbol, FindReferencesDocumentState, SyntaxToken, CancellationToken, ValueTask<(bool matched, CandidateReason reason)>> GetParameterSymbolsMatchFunction(
```

Comment:
this got substantively changed.  instead of having a single parameter trying to match against lots of different symbols, instead we cascade first to get the different symbols, and then do matching normally.